peep.py

```diff
@@ -727,8 +731,18 @@ def downloaded_reqs_from_path(path, argv):
     :arg argv: The commandline args, starting after the subcommand
 
     """
-    return [DownloadedReq(req, argv) for req in
-            parse_requirements(path, options=EmptyOptions())]
+    try:
+        return [DownloadedReq(req, argv) for req in
+                parse_requirements(path, options=EmptyOptions())]
+    except TypeError:
```

Comment:
No, I believe you could import PipSession from pip.download before `session` became a required kwarg, I'm just not sure exactly what version of pip parse_requirements began accepting the `session` kwarg, so it seemed safer to use the original call that we knew worked up to pip 1.5.6 and only pass in a session if it complains with the TypeError.